frontend/ui/widget/booklist.lua

```diff
@@ -103,15 +103,17 @@ function BookList.getBookStatus(file)
 end
 
 function BookList.getBookStatusString(status, with_prefix)
-    local status_string = ({
-        new       = _("New"),      -- no sidecar file
-        reading   = _("Reading"),  -- doc_settings.summary.status
-        abandoned = _("On hold"),  -- doc_settings.summary.status
-        complete  = _("Finished"), -- doc_settings.summary.status
-        deleted   = _("Deleted"),
-        all       = _("All"),
-    })[status]
-    return with_prefix and T(_("Status: %1"), status_string:lower()) or status_string
+    if status then
+        local status_string = ({
+            new       = _("New"),      -- no sidecar file
+            reading   = _("Reading"),  -- doc_settings.summary.status
+            abandoned = _("On hold"),  -- doc_settings.summary.status
+            complete  = _("Finished"), -- doc_settings.summary.status
+            deleted   = _("Deleted"),
+            all       = _("All"),
+        })[status]
```

Comment:
(You're just moving it around, so I should have asked that ealier :)
Not worth having it as a module local ?
(Dunno how luajit works with that, but creating that small table everytime, with newly created strings then removed, may uses more cpu than if it was long-living - moreover if this function BookList.getBookStatusString() can be caller tons of time when we browse files or collections.)

Reply:
Done